### PR TITLE
IGVF-1191-fileset-page-ui

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: components/common-data-items.js [hunk 1]
```diff
@@ -602,6 +602,75 @@ FileDataItems.commonProperties = [
...
+            <DbxrefList dbxrefs={item.dbxrefs} />
+          </DataItemValue>
+        </>
+      )}
+    </>
+  );
+}
+
+FileSetDataItems.propTypes = {
+  // file object common for all file types
```
Comment (forresttanaka) [inline components/common-data-items.js:661]: Really “FileSet object common for all FileSet types.”

File: pages/auxiliary-sets/[id].js [hunk 2]
```diff
@@ -142,8 +131,7 @@ export default function AuxiliarySet({
                   <DataItemValue>{auxiliarySet.moi}</DataItemValue>
                 </>
               )}
-              <DataItemLabel>Summary</DataItemLabel>
-              <DataItemValue>{auxiliarySet.summary}</DataItemValue>
+              <FileSetDataItems item={auxiliarySet} />
```
Comment (forresttanaka) [inline pages/auxiliary-sets/[id].js:134]: This comment applies to all the FileSet pages. I think you want to wrap the type-specific data items within `FileSetDataItems`. See how `BiosampleDataItems` wraps all the InVitroSystem-specific data items within itself. All the data items it wraps goes into the `children` property of `FileSetDataItems`.

**Review by forresttanaka — CHANGES_REQUESTED** [review]

File: components/common-data-items.js (hunk 1 above)
Comment (khzlin) [reply components/common-data-items.js:661]: done

File: pages/auxiliary-sets/[id].js (hunk 2 above)
Comment (khzlin) [reply pages/auxiliary-sets/[id].js]: done

**Review by forresttanaka — APPROVED** [review]

**Pull request merged**